sbt-metals/src/main/scala/scala/meta/sbt/MetalsPlugin.scala

```diff
@@ -117,13 +124,8 @@ object Metals {
       "Generates .metals/buildinfo/**.properties files containing build metadata " +
         "such as classpath and source directories.",
     detail = ""
-  ) { s =>
-    val configDir = s.baseDir / ".metals" / "buildinfo"
-    IO.delete(configDir)
```

Comment:
This is the best place to delete `.metals/buildinfo` IMO

Reply:
The command should be just a shortcut and do as less as possible. So it just calls `*/melastWriteBuildInfo`, which [does this cleanup](https://github.com/scalameta/metals/pull/269/files#diff-aa395a3cb94cc4303b23c0bfb833febcR68).